app/pkg/config/config.go

```diff
@@ -1,38 +1,38 @@
 package config
 
 import (
-  "encoding/json"
-  "fmt"
-  "io/fs"
-  "os"
-  "os/user"
-  "path/filepath"
-  "strings"
-  "time"
-
-  "github.com/jlewi/cloud-assistant/app/api"
-
-  "github.com/go-logr/zapr"
-  "github.com/pkg/errors"
-  "github.com/spf13/cobra"
-  "github.com/spf13/viper"
-  "go.uber.org/zap"
-  "gopkg.in/yaml.v3"
+	"encoding/json"
```

Comment:
I wonder why we keep getting these large whitespace changes in golang and why they aren't being caught by our lint checks.

Reply:
My guess is that it's a Goland <> VS Code's Go extension mismatch? I'm gonna open a PR to add pre-commit hooks to enforce consistency. I agree, these whitespace issues make diffs difficult.